src/service/repo/anonymousMessageRepo.ts

```diff
@@ -34,7 +34,11 @@ interface AnonymousMessageRepoTypes {
   getAllAnonymousPostNotifications: (timeStamp: string) => Promise<AnonymousPostNotification[]>;
   getSingleAnonymousPostNotifications: (activityId: string) => Promise<AnonymousPostNotification>;
   setChannelAsRead: (channelId: string) => Promise<boolean>;
-  getAnonymousChannelDetail: (channelType: GetstreamChannelType, channelId: string) => Promise<any>;
+  getAnonymousChannelDetail: (
+    channelType: GetstreamChannelType,
+    channelId: string,
+    timeStamp?: string
+  ) => Promise<any>;
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of the `timeStamp` parameter to the `getAnonymousChannelDetail` function is a good change. It allows for more flexibility in fetching channel details based on a specific timestamp. However, it would be beneficial to add some comments explaining what this new parameter does and how it affects the function's behavior.



<!-- This is an auto-generated comment by OSS CodeRabbit -->